lib/api/app_worker/element_handler.dart

```diff
@@ -55,6 +56,18 @@ mixin ElementHandler<M> on ServiceWorker<M>, StopAreaHandler<M>, MapFeatureHandl
 
   final _osmElementQueryHandler = OSMElementQueryAPI();
 
+  @override
+  void updateQuestionCatalog(({QuestionCatalog questionCatalog, bool onlyLanguageChange}) questionCatalogChangeData) async {
+    super.updateQuestionCatalog(questionCatalogChangeData);
+    final mfCollection = await mapFeatureCollection;
+    final existingElements = _filterElements(
+      _buildFiltersForStopAreas(loadedStopAreas),
+      Stream.fromIterable(_elementPool.elements),
+    );
+    _elementStreamController.add(const ElementUpdate(element: null, action: ElementUpdateAction.clear));
+    existingElements.map((element) => ElementUpdate.derive(element, mfCollection, action: ElementUpdateAction.update))
+    .forEach(_elementStreamController.add);
```

Comment:
Prefer formatting like this

    existingElements
      .map((element) => ElementUpdate.derive(element, mfCollection, action: ElementUpdateAction.update))
      .forEach(_elementStreamController.add);